### DISCUSSION TITLE
self hosted runner stuck on queued

### DISCUSSION THREAD

**Discussion by tdubb123** [actions/runner#2457] [General]
i got self hosted runner on prem but its stuck in queued

Waiting for a runner to pick up this job...
Job is about to start running on the runner: k8gui (repository)

any idea? 

my workflow file is

name: remote ssh command
on: [push]
jobs:

  build:
    name: Build
    runs-on: self-hosted
    steps:
    - name: executing remote ssh commands using password
      uses: appleboy/ssh-action@v0.1.6
      with:
        host: ${{ secrets.HOST }}
        username: ${{ secrets.USERNAME }}
        script: whoami

from the runner vm, I can ssh to HOSt without password

**Comment by joebnb** [comment]
same problem，i dont know is the ssh action have bug or runner have bug, but it's cant use github runner to login my local machine

**Comment by joebnb** [comment]
did you solve this problem？
